src/tools/annotation/EllipticalRoiTool.js

```diff
@@ -355,7 +355,7 @@ function _getUnit(modality, showHounsfieldUnits) {
 function _createTextBoxContent(
   context,
   isColorImage,
-  { area, mean, stdDev, min, max, meanStdDevSUV } = {},
+  { area = 0, mean = 0, stdDev = 0, min = 0, max = 0, meanStdDevSUV = 0 } = {},
```

Comment:
Change to


context,
  isColorImage,
  ellipticalContent = {}
  
  const { area = 0, mean = 0, stdDev = 0, min = 0, max = 0, meanStdDevSUV = 0 } = ellipticalContent